### PR TITLE
Add regression test for #1213, Access modifier is lost in extern declaration

### PR DESCRIPTION
**Pull request by dawedawe**
#1213 is already fixed in current. So here is a regression test to close the issue.

### CODE REVIEW

**Review by nojaf — CHANGES_REQUESTED** [review]
Thanks again for this PR.
One minor nitpick.

File: src/Fantomas.Tests/SignatureTests.fs [hunk 1]
```diff
@@ -207,6 +207,23 @@ type Test =
     static member internal FormatAroundCursorAsync: fileName: string -> unit
 """
 
+[<Test>]
```
[the remark this answers is not in the document]
Comment (dawedawe) [reply src/Fantomas.Tests/SignatureTests.fs:210]: Ups, sorry.

[the remark this answers is not in the document]
Comment (dawedawe) [reply src/Fantomas.Tests/SignatureTests.fs]: Sorry. I fucked up my clone.

**Review by nojaf — APPROVED** [review]
Thanks!

**Pull request merged**